### DISCUSSION TITLE
Loco management.

### DISCUSSION THREAD

**Discussion by Budman1758** [DCC-EX/EX-T3-WiFi#65] [Q&A] [answered]
As usual, I'm a bit confused here. Trying to set up functions for custom loco commands and getting nowhere. I'm thinking that there is something missing on this page. There is nothing on the page to "drag in". I uploaded the file system B4 uploading the program to the throttle and the icons are uploaded also. Not sure where to go here. Any help for the helpless helps. Thanks.

![image](https://user-images.githubusercontent.com/6570134/220489667-fdae4d53-5b06-4fb7-b36a-08a38bc8a60b.png)

**Comment by alex-code** [comment, marked as answer]
The first blue button below the black rectangle is for rows, the middle one is for function buttons.
You start by dragging in a row then dragging buttons into the row.

https://github.com/DCC-EX/EX-T3-WiFi/blob/master/docs/locos.md#custom-functions
